Directory.Build.props

```diff
@@ -32,10 +32,6 @@
       NuGet: https://www.nuget.org/packages/coverlet.collector/
     -->
     <CoverletCollectorVersion>3.1.2</CoverletCollectorVersion>
-    <!--
-      NuGet: https://www.nuget.org/packages/Swashbuckle.AspNetCore/
-    -->
```

Comment:
This _might_ have been in here to help fascilitate SDK-API integration @Hinton, do we still need this?